mobile/EmbeddedCounselDemo/EmbeddedCounselDemo/AccountView.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AccountView.swift
+//  EmbeddedCounselDemo
+//
+//  Created by Counsel on 5/13/25.
+//
+
+import SwiftUI
+
+struct AccountView: View {
+
+    @Binding var presentAccessCodeModal: Bool
+    @AppStorage("token") private var token: String?
+
+    var body: some View {
+        NavigationStack {
+            VStack {
+                Spacer()
+                Button(action: {
+                    token = nil // clear token
```

Comment:
This should call the `/user/signOut` API on the demo server

Reply:
added